packages/devtools_app/lib/src/shared/framework/screen.dart

```diff
@@ -47,6 +47,7 @@ enum ScreenMetaData {
     title: 'Performance',
     iconAsset: 'icons/app_bar/performance.png',
     worksWithOfflineData: true,
+    requiresWebAppDebuggable: true,
```

Comment:
I find the name confusing - `requiresWebAppDebuggable` makes it sound like the panel is only available for web apps, when in fact the performance panel is not supported for web apps but is shown because it has instructions for how to use performance profiling in Chrome DevTools. 

I would suggest `supportsWebServerDevice` to make this more clear, with the default value being `false` (so we only need to add `supportsWebServerDevice` on the few screens we want to support here)

Reply:
Done!